server/vespa/reindex.sh

```
@@ -0,0 +1,80 @@
+
+vespa prepare
+vespa activate
+
+IFS=',' read -ra docTypes <<< "$documentTypes"
+while true; do
+    STATUS=$(curl -s "http://localhost:19071/application/v2/tenant/default/application/default/environment/default/region/default/instance/default/reindexing")
+    all_successful=true
+    any_error=false
+    
+    # Check status for each document type
+    for docType in "${docTypes[@]}"; do
```

Shellcheck (0.10.0)

[warning] In POSIX sh, array references are undefined.

(SC3054)